README.md

```diff
@@ -1,30 +1,96 @@
-## Synopsis
+# FreeOrion
 
-Source code repository of the FreeOrion project.
+FreeOrion is a free, Open Source, turn-based space empire and galactic conquest
+computer game being designed and built by the FreeOrion project.
 
-FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game. 
+Freeorion builds on the classic *4X* model by incorporating the nation-building
+elements of games such as *Europa Universalis 2*.  While its modular,
+Open Source design allows for a significant degree of customization of the game
+engine and the story elements by the community, the FreeOrion project is
+dedicated to the construction of a living, breathing universe in a
+*grand campaign* model.
 
-## Vision Statement
 
-FreeOrion, an open-source game inspired by Master of Orion, is a turn-based game of epic space strategy that builds on the classic '4X' model by incorporating the nation-building elements of games such as Europa Universalis 2 and a versatile tactical combat engine. While its modular, open-source design allows for a significant degree of customization of the game engine and the story elements by the community, the FreeOrion team is dedicated to the construction of a living, breathing universe in a 'grand campaign' model.
+## Requirements
 
-## How to contribute
+FreeOrion requires an *OpenGL 2.1* capable graphic card and a display with a
+minimum resolution of at least *800x600*.
 
-For more details please see our [Contribution Guidelines](CONTRIBUTING.md).
+FreeOrion requires at least *Windows XP* with *Service Pack 2* or later,
+*Mac OSX 10.7* or later or any reasonable recent Linux distribution on x86
```

Comment:
I agree with you in general here, but ignore this comment for now, as it is off topic for the PR. The fact itself was a copy and paste from the wiki requirements page and I noticed that the wiki is totally inconsistent.  We should remove all those pages and link instead to the README.md in the future.